docs/guides/snowflake_integration.md

```diff
@@ -486,3 +505,128 @@ FN_T_UID2_IDENTITY_MAP(EMAIL_HASH, 'email_hash')
 When you have the new function implemented, you can check the `UNMAPPED` column returned by the `FN_T_UID2_IDENTITY_MAP`. If any DII could not be mapped to a UID2, this column gives the reason.
 
 For details about the values and their explanations, see [Values for the UNMAPPED Column](#values-for-the-unmapped-column).
+
+## Usage for UID2 Sharers
+
+A UID2 sharer is any participant that wants to share UID2s with another participant. <!-- For details, see [UID2 Sharing Overview](../sharing/sharing-overview). -->
+
+A sharing participant must encrypt [Raw UID2s](../ref-info/glossary-uid#gl-raw-uid2) into [UID2 tokens](../ref-info/glossary-uid#gl-uid2-token) before sending them to another participant.
+
+The following functions support UID2 sharing:
+
+- [FN_UID2_ENCRYPT](#fn_uid2_encrypt)
+- [FN_UID2_DECRYPT](#fn_uid2_decrypt)
+
+### FN_UID2_ENCRYPT
+
+This function encrypts a raw UID2 and returns the corresponding UID2 token.
+
+Use the applicable prefix to indicate your role:
+- For advertisers: `ADV.FN_UID2_ENCRYPT`
+- For data providers: `DP.FN_UID2_ENCRYPT`
+
+>NOTE: The raw UID2 value is a Base64-encoded string, 44 characters in length.
+
+#### FN_UID2_ENCRYPT: Syntax
+
+The following is the basic syntax for the encrypt function:
+
+```
+Function: FN_UID2_ENCRYPT(raw_uid VARCHAR(44))
+RETURNS VARCHAR(220)
+```
+
+#### FN_UID2_ENCRYPT: Example
+
+The following example illustrates use of this function. The actual change varies according to the specific database, so the `WHERE` is not specified in the example.
+
+```
+SELECT T.token, LATERAL (FN_UID2_DECRYPT(T.token)) AS raw_uid
+FROM receiver's_table T
```

Comment:
`receiver's_table` isn't a valid table name. In other examples we use table names like `AUDIENCE` or `AUDIENCE_WITH_UID2`. Can we use something like that instead of `receiver'_table` and `sender's_table`?

Reply:
TY! Fixed in all examples plus modified the lead-in to clarify. Please check again.